ansible_collections/arista/avd/plugins/module_utils/eos_designs_facts.py

```diff
@@ -705,6 +705,51 @@ def evpn_gateway_remote_peers(self):
             return get(self._switch_data_combined, "evpn_gateway.remote_peers")
         return None
 
+    @cached_property
+    def mpls_vpn_gateway(self):
```

Comment:
Instead of adding 9 new facts for one rarely used feature, I think it would be much simpler to just add a single `mpls_vpn_gateway`. You can implement the defaults in that function, and return a dict with settings(or defaults) set as expected.

Reply:
Fixed this. Added a dict instead.